nifi-nar-bundles/nifi-cassandra-bundle/nifi-cassandra-processors/src/main/java/org/apache/nifi/processors/cassandra/QueryCassandra.java

```diff
@@ -130,6 +132,24 @@
             .defaultValue(AVRO_FORMAT)
             .build();
 
+    public static final PropertyDescriptor DATE_FORMAT_PATTERN = new PropertyDescriptor.Builder()
+            .name("timestamp-format-pattern")
```

Comment:
Minor: I think there is only a small chance for having different "timestamp format pattern" property other than the one for JSON, but as property names should not change later, I would suggest to keep the name consequent to the display name and involve json in it.

Reply:
Thanks @simonbence , I have updated the PR according to your suggestion.